src/components/progress/Progress.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="progress-wrapper">
+        <progress
+            ref="progress"
+            class="progress"
+            :class="newType"
+            :max="max">{{ newValue }}</progress>
+        <p
+            v-if="showValue"
+            class="progress-value has-text-white"><slot>{{ newValue }}</slot></p>
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'BProgress',
+    props: {
+        type: {
+            type: String | Object,
```

Comment:
Is a really correct syntax ? I usually use [String, Object] array syntax.

Reply:
Your syntax is more common for Vue props. I'll fix it!